javascript/javascript1/week1/index.js

```diff
@@ -0,0 +1,70 @@
+//Age-ify (A future age calculator)
+const yearOfBirth = 1987;
+const yearFuture = 2027;
+console.log(yearFuture - yearOfBirth);
+
+const age = 40;
+console.log(
+  "You will be" + " " + age + " " + "years old in" + " " + yearFuture + "."
+);
+
+//Goodboy-Oldboy (A dog age calculator)
+
+const dogYearOfBirth = 1957;
+const dogYearFuture = 2027;
+console.log(dogYearFuture - dogYearOfBirth);
+
+const dogYear = 70;
+
+if (dogYear < 70) {
+  console.log(
+    "Your dog will be " + dogYear + " human years old in " + dogYearFuture
+  );
+} else {
+  console.log(
+    "Your dog will be " + dogYear + " dog years old in " + dogYearFuture
+  );
+}
+
+//Housey pricey (A house price estimator).
+
+//Peter
+let housePrice = 2.5;
+let volumeInMeters = 8 * 10 * 10;
+let gardenSizeInM2 = 100;
+
+console.log((housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300));
+let costOfHouse = 2.03;
+
+//Julia
+housePrice = 1.0;
+volumeInMeters = 5 * 11 * 8;
+gardenSizeInM2 = 70;
+
+console.log((housePrice = volumeInMeters * 2.5 * 1000 + gardenSizeInM2 * 300));
+costOfHouse = 1.121;
+
+/*if (housePrice > costOfHouse) {
+  console.log("paid too much");
+}*/
+if (housePrice < costOfHouse) {
+  console.log("paid less");
+} else {
+  console.log("paid too much");
+}
+
+//Ez Namey (Startup name generator)
+
+let firstWords = ["Mega", "Awesome", "Flash", "Otto", "Andrew"];
+let secondWords = ["Comp", "Java", "Point", "It", "ltd"];
+let startupName = firstWords.concat(secondWords);
+console.log(startupName);
+startupName.length;
+
+/*I did not understand this other part of the exercise*/
```

Comment:
That's ok - it looks like you tried! 

You were meant generate two random numbers between 0 and the length of your words array -1; in this case it would be 4. Using this number we could access a word from each array, concatenate them together, and create a startup name.  

A potential solution could look like this: 
```js
const firstWords = ["Mega", "Awesome", "Flash", "Otto", "Andrew"];
const secondWords = ["Comp", "Java", "Point", "It", "ltd"];

const randomNumber1 = Math.floor(Math.random() * firstWords.length);
const randomNumber2 = Math.floor(Math.random() * secondWords.length);

const startupName = firstWords[randomNumber1] + " " + secondWords[randomNumber2]

console.log(starupName)
```

Spend some time reading this through - I think it could help with your understanding of working with arrays: https://www.digitalocean.com/community/tutorials/understanding-arrays-in-javascript